verification/curator-service/api/src/controllers/cases.ts

```diff
@@ -0,0 +1,24 @@
+import { Request, Response } from 'express';
+import axios from 'axios';
+
+export interface Case {
```

Comment:
should we start a common directory or something where we share those interfaces?
Or should we keep those separate despite the duplication? I can see pros and cons to both...

Reply:
Yeah, I can see both arguments. I don't know whether we should publish the Mongoose document type, but I do think it'd be reasonable for the data-service to publish a "public" dir/package that contains the type, in the least.

Seems like a common problem in Node land; I'll do some research.